.github/workflows/vrt.yml

```diff
@@ -25,7 +32,8 @@ permissions:
 
 jobs:
     vrt:
-        if: ${{!startsWith(github.ref, 'refs/heads/dependabot/') && !startsWith(github.ref, 'refs/heads/changeset-release/') }}
+        # if the branch being merged is a dependabot branch or a changeset release, skip everything in this workflow
+        if: ${{!startsWith(github.head_ref, 'dependabot') && !startsWith(github.head_ref, 'changeset-release') }}
```

Comment:
I may need to also check that it was `closed` due to a `merge`, because this might run for anything that is closed without merging.

Reply:
Won't this add additional runs on closed PRs instead of reducing the number of runs against main?

Reply:
Yes! I need to exclude it from regular closures instead of "closures that are merges".